packages/frontend/src/pages/Activate/ActivatePage.tsx

```diff
@@ -13,11 +13,12 @@ import { useRecoilValue } from 'recoil';
 import ActivateButton from './components/ActivateButton';
 import EthAccount from './components/EthAccount';
 
-const hasMetaMask = () => {
+const hasMetaMask = (): boolean => {
```

Comment:
Maybe `hasMetaMask` should be pulled out into utils since it's defined in 2 components.

Reply:
✅